### ISSUE TITLE
Is anyone mocking Mongo, Elasticsearch or similar?

### ISSUE THREAD

**Issue by andrewmclagan** [typicode/json-server#468]
Is anyone using this package to mock-out an Elasticsearch instance or a MongoDB instance?

THAT would be very interesting.

**Comment by jacobwarduk** [comment]
Currently working on trying to mock out Elasticsearch using json-server at work. It's not been a fun day so far!

**Comment by jacobwarduk** [comment]
@andrewmclagan Yeah, we've basically gotten nowhere with it. So I wouldn't get your hopes up.

Though if you do find something, please do let me know!

**Comment by fega** [comment]
maybe this could be interesting: https://github.com/fega/mongo-server
